### ISSUE TITLE
CMake highlighting not works for KDE related keywords

### ISSUE THREAD

**Issue by Anteru** [pygments/pygments-migration-test#573]
(Original issue 866 created by maidis on 2013-04-22T21:47:20.949066+00:00)

**KDE** has some special CMake commands like *kde4_add_executable* and these not recognized by pygments in Sphinx.

I don't know if this is the right solution but when I changed following line:

```
#!python

(r'\b([A-Za-z_]+)([ \t]*)(\()', bygroups(Name.Builtin, Text,
```

with this one:

```
#!python

(r'\b([A-Za-z0-9_]+)([ \t]*)(\()', bygroups(Name.Builtin, Text,
```

everything started to work properly

Some related links:

* [CMake Highlighting with literalinclude](https://groups.google.com/forum/?fromgroups=#!topic/sphinx-users/dQu3vkVPCq8%5B1-25-false%5D)

* [CMake dosyaları renklendirilmiyor](https://github.com/maidis/kde-gelistirme-kitabi/issues/1)

**Comment by Anteru** [comment]
(Original issue 866 last updated on 2013-05-19T09:37:01.182158+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by Anteru** [comment]
(Original comment by birkenfeld on 2013-05-19T09:37:01.175684+00:00)

Closes #866: allow digits in CMake directives.

→ <<cset cd1cfa7df9f2>>
